### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by mbroz**

### CODE REVIEW

**Comment by mbroz** [comment]
@taramka check this please

File: rng.c [hunk 1]
```diff
@@ -20,10 +20,10 @@ enum arg_keys
 static struct argp_option options[] = {
     { "bytes", ARG_KEY_REQ_BYTES, "number of bytes", 0, "Number of random bytes within a sequence (defaults to 32 bytes)", 0 },
     { "chunk-size", ARG_KEY_CHUNK_SIZE, "size of chunks in bytes", 0, "Divide acquisition of the random sequence into chunks of a specified size (defaults to 512 bytes)", 0 },
-    { "output", ARG_KEY_OUTPUT_FILE, "file", 0, "Output file (defaults to stdout)", 0 },
+    { "output", ARG_KEY_OUTPUT_FILE, "file", 0, "Output file, use \"-\" for binary stdout output", 0 },
     { "log-level", ARG_KEY_LOG_LEVEL, "level", 0, "Log level", 0 },
     { "scsi", ARG_KEY_USE_SCSI, 0, 0, "Use SCSI", 0 },
-    { "hex", ARG_KEY_HEX_OUTPUT, 0, 0, "Output random bytes in hexadecimal (NOT IMPLEMENTED)", 0 },
+    { "hex", ARG_KEY_HEX_OUTPUT, 0, 0, "Output random bytes in hexadecimal", 0 },
```
Comment (taramka) [inline rng.c:26]:
```suggestion
    { "hex", ARG_KEY_HEX_OUTPUT, 0, 0, "Output random bytes in hexadecimal, only available for stdout output", 0 },
```

File: rng.c [hunk 2]
```diff
@@ -20,10 +20,10 @@ enum arg_keys
 static struct argp_option options[] = {
     { "bytes", ARG_KEY_REQ_BYTES, "number of bytes", 0, "Number of random bytes within a sequence (defaults to 32 bytes)", 0 },
     { "chunk-size", ARG_KEY_CHUNK_SIZE, "size of chunks in bytes", 0, "Divide acquisition of the random sequence into chunks of a specified size (defaults to 512 bytes)", 0 },
-    { "output", ARG_KEY_OUTPUT_FILE, "file", 0, "Output file (defaults to stdout)", 0 },
+    { "output", ARG_KEY_OUTPUT_FILE, "file", 0, "Output file, use \"-\" for binary stdout output", 0 },
```
Comment (taramka) [inline rng.c:23]:
```suggestion
    { "output", ARG_KEY_OUTPUT_FILE, "file", 0, "Output file for binary output, use \"-\" for binary stdout output (defaults to stdout hexadecimal output)", 0 },
```

**Review by taramka — APPROVED** [review]
I suggested two small additions to the `--help` output according to the new program behavior; ignore, apply or modify them to your liking.

Looks good to me otherwise and seems to work fine too.

**Comment by mbroz** [comment]
Fixed, also with small change for clang compilation and warning.

**Pull request merged**